.github/workflows/scripts/ti_build/llvm.py

```diff
@@ -44,4 +48,8 @@ def setup_llvm() -> None:
 
     # We should use LLVM toolchains shipped with OS.
     # path_prepend('PATH', out / 'bin')
-    os.environ["LLVM_DIR"] = str(out)
+    if (u.system, u.machine) not in (("Linux", "arm64"), ("Linux", "aarch64")):
```

Comment:
Please could you explain
1. what the original intent of this line is?
2. why does the new behavior vary for arm vs x86?

Reply:
Because not exists assets right now on that links for arm

Reply:
Yeah, we need an arm build of llvm right? Shall I create an empty repo for an llvm build, and you can PR an llvm build script onto that repo?